Userland/Libraries/LibLine/Editor.cpp

```diff
@@ -560,7 +560,7 @@ void Editor::initialize()
             m_configuration.set(Configuration::NonInteractive);
         } else {
             auto* term = getenv("TERM");
-            if (term != NULL && StringView { term, strlen(term) }.starts_with("xterm"sv))
+            if ((term != NULL) && StringView { term, strlen(term) }.starts_with("xterm"sv))
```

Comment:
these are interesting. I thought we didn't really have the style to wrap compound boolean expressions in extra parens. I don't mind that style tbh, it's more explicit.

Reply:
Ah I mean to add a comment to the PR after opening it for these. Clang-format-16 has a bug that pushes the `&&` one space to the left now, so this would look like:
```c++
if (term != NULL&& StringView { term, strlen(term) }.starts_with("xterm"sv))
```

I think it has to do with the inlined `{ }` style constructor, and I'll ping Emilia about it tomorrow. But putting parens around the inner statement prevents this for now.